demisto_sdk/tests/test_files/script-valid.yml

```diff
@@ -15,6 +15,7 @@ args:
 scripttarget: 0
 runonce: false
 fromversion: 5.0.0
+engineinfo: ""
```

Comment:
It's tested [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/validate/tests/validators_test.py#L620)